mesonbuild/build.py

```diff
@@ -3090,6 +3090,9 @@ def extract_all_objects(self) -> T.List[T.Union[str, 'ExtractedObjects']]:
     def get_custom_install_dir(self) -> T.List[T.Union[str, Literal[False]]]:
         return self.target.get_custom_install_dir()
 
+    def get_option(self, key: 'OptionKey') -> T.Union[str, int, bool, T.List[str]]:
```

Comment:
Why is OptionKey quoted here ?

Reply:
Copy and past. Fixed in the latest version